app/views/compute_resources/form/_ec2.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= select_f(f, :http_proxy_id, HttpProxy.all, :id, :name,
```

Comment:
Nitpick: Maybe move this to a helper to allow for easier inclusions by other compute resources?